build-functions.php

```diff
@@ -29,7 +29,7 @@ function kebabCaseToUcWords($kebab)
  */
 function toKebabCase($string)
 {
-    return implode('-', explode(' ', strtolower($string)));
+    return preg_replace('/[ _\\-\\+\\/\\\\]+/', '-', strtolower($string));
```

Comment:
This is wrong. See [example](http://www.phpliveregex.com/p/kCA). You should probably first replace all separation chars, e.g. `- +_/\\`, with `-`, and then collapse all collapsible chars similarly to what you did, but capture the actual char, and replace with a single instance of that char, e.g. `$0`.